packages/cozy-client/src/models/file.js

```diff
@@ -622,6 +622,68 @@ export const fetchBlobFileById = async (client, fileId) => {
   return fileBlob
 }
 
+/**
+ * @param {IOCozyFile} file - io.cozy.files document
+ * @returns {boolean}
+ */
+const isExpiringFrenchNationalIdCard = file => {
+  const label = get(file, 'metadata.qualification.label', null)
+  const country = get(file, 'metadata.qualification.country', null)
+  const expirationDate = get(file, 'metadata.expirationDate', null)
+  if (
+    label === 'national_id_card' &&
```

Comment:
We should not put this check at the same level as the `expirationDate`. 

Either we make sure that we can't pass wrong data here or we can do something like: 
```
const isExpiringFrenchNationalIdCard = file => {
  const label = get(file, 'metadata.qualification.label', null)
  const country = get(file, 'metadata.qualification.country', null)
  const expirationDate = get(file, 'metadata.expirationDate', null)
if(label !== 'national_id_card'){
logger.log(`You called isExpiringFrenchNationalIdCard for a non national id card. File: ` JSON.stringify(file)`)
}
if(country && country !== 'fr'){logger.log... }

Reply:
The goal of this function is not to test if a given national id card is french with an expiration date, but instead if a given file is a national id card that is French and with an expiration date.
Being able to be called on any file is on purpose.

Reply:
This is not clear. You should add it to the description.

Reply:
I added a description to each function that should hopefully clarify that.